examples/github_app_authentication.rs

```diff
@@ -0,0 +1,48 @@
+use octocrab::models::{CreateInstallationAccessToken, Installation, InstallationToken};
+use octocrab::{create_authenticate_as_app_jwt, Octocrab};
+
+#[tokio::main]
+async fn main() -> octocrab::Result<()> {
+    let app_id = read_env_var("GITHUB_APP_ID");
+    let app_private_key = read_env_var("GITHUB_APP_PRIVATE_KEY");
+
+    let octocrab = Octocrab::builder()
+        .personal_token(create_authenticate_as_app_jwt(&app_id, &app_private_key).unwrap())
+        .build()?;
+
+    let installations: Vec<Installation> = octocrab
+        .get("/app/installations", None::<&()>)
+        .await
+        .unwrap();
+
+    let mut create_access_token = CreateInstallationAccessToken::default();
+    create_access_token.repositories = vec!["octocrab".to_string()];
+
+    let access: InstallationToken = octocrab
```

Comment:
This fails to deserialize. Looks like the returned Permissions struct is missing the `admin` field.

Can I add `#[serde(default)]` to the admin field in the `Permissions` struct?

Reply:
Done